File: src/main/java/com/tournament/app/footycup/backend/dto/team/TeamStatisticsResponse.java
package com.tournament.app.footycup.backend.dto.team;

public record TeamStatisticsResponse(
        Long teamId,
        String teamName,
        Integer matchesPlayed,
        Integer minutesPlayed,
        Integer goals,
        Integer yellowCards,
        Integer redCards,
        Integer substitutions,
        Integer otherEvents,
        Integer totalEvents
) {
}

File: frontend/src/app/models/team/team-statistics.response.ts
export interface TeamStatisticsResponse {
  teamId: number;
  teamName: string;
  matchesPlayed: number;
  minutesPlayed: number;
  goals: number;
  yellowCards: number;
  redCards: number;
  substitutions: number;
  otherEvents: number;
  totalEvents: number;
}